javascript/javascript2/week2/doublingOfNumbers.js

```diff
@@ -0,0 +1,11 @@
+
+function getDoubleOddNumber () {
+  let result = doubledOddNumber;
+  return result
+}
+
+const numbers = [1, 2, 3, 4];
+const oddNumbers = numbers.filter (number => number % 2);
+let doubledOddNumber = oddNumbers.map(x => x*2)
```

Comment:
Since `doubleOddNumber` is never reassigned, you should use `const`. Just like you did with the other variables 👍 

Since `doubleOddNumber` is an array, you could consider using the plural name `doubleOddNumbers` with an s.
Also, remember the semicolon :)